packages/cw-orch-mock/src/state.rs

```diff
@@ -21,8 +23,26 @@ impl MockState {
         Self {
             addresses: HashMap::new(),
             code_ids: HashMap::new(),
+            chain_id: mock_env().block.chain_id,
```

Comment:
chain-id is set in `cw_multi_test::App`, how does this relate to that? 

This could (and probably will) cause issues in the future

Reply:
Yes, however, the Deploy Details trait necessitates the chain_id field, see below in the same file. It's not a perfect solution, yes !

Reply:
So for this we discussed removing this chain_id and updating the `deploy_info` function to use a seperate trait (like `EnvironmentInfo`) that we implement for each env and add as a traitbound to `CwEnv`.